app/controllers/admin/adjustments_controller.rb

```diff
@@ -0,0 +1,49 @@
+class Admin::AdjustmentsController < ApplicationController
+  load_permissions_and_authorize_resource
```

Comment:
This controller is only interacted by admins, so we could remove the `before_action`